FHIR/src/Microsoft.Health.Fhir.Anonymizer.Shared.Core/AnonymizerEngine.cs

```diff
@@ -109,6 +109,14 @@ public string AnonymizeJson(string json, AnonymizerSettings settings = null)
             return anonymizedElement.ToJson(serializationSettings);
         }
 
+        public void AddCustomProcessors(string methodKey, IAnonymizerProcessor processor)
```

Comment:
Can we add unit tests for the FHIR project as well?

Reply:
Thanks, added it.